portfolio/src/main/webapp/index.html

```diff
@@ -344,7 +344,7 @@ <h3 class="section-subheading text-muted">Click on each picture to learn more.</
     </footer>
     <!-- Martyr Modals-->
     <!-- TODO(briafassler/kofimeighan): finish inserting information for interactive pictures-->
-    <!-- Modal 1-->
+    <!-- Modal 1: George Floyd-->
```

Comment:
+1. These changes were made in my previous PR requests.

Reply:
It is diffing against the wrong base branch. I have the base of this PR on 'Comments' because a lot of the work done here is based on the things I coded within that PR.